lms/djangoapps/instructor/management/commands/dump_userinfo.py

```diff
@@ -291,6 +342,31 @@ def handle(self, *args, **options):
         outfile.close()
         sys.stdout.write("Data written to {name}\n".format(name=outfile_name))
 
+    def build_new_columns(self, user, course_id):
```

Comment:
two lines before function declarations...

Reply:
We went by: method definitions inside a class are surrounded by a single blank line.

Reply:
Since build_new_columns it's a method of the Command class, has one blank line according to https://www.python.org/dev/peps/pep-0008/#blank-lines

Reply:
yep you're right, sorry.